src/components/custom-field-input-single-choice/custom-field-input-single-choice.test.jsx

```diff
@@ -13,17 +13,31 @@ describe('src/components/custom-field-input-single-choice/custom-field-input-sin
   });
 
   describe('accessibility', () => {
+    const choices = [{
+      id: 'foo',
+      label: 'foo',
+    }, {
+      id: 'bar',
+      label: 'bar',
+    }];
+
     it('shows choices when clicked', () => {
-      const choices = ['foo', 'bar'];
       render(<CustomFieldInputSingleChoice label="Foo" id="bar" choices={choices} />);
       userEvent.click(screen.getByLabelText('Foo'));
 
       expect(screen.getByText('foo')).toBeInTheDocument();
       expect(screen.getByText('bar')).toBeInTheDocument();
     });
 
+    it('does not show choices when readOnly is true', () => {
```

Comment:
Is this a better test for the `readOnly` API? (Why do we think it is an accessibility test?)